public/images/appstore-dark.svg

```diff
@@ -0,0 +1 @@
8.7-24 22.2-24 13.6 0 22.2 9.3 22.2 24 0 14.8-8.6 24-22.2 24s-22.2-9.2-22.2-24zm34.6 0c0-10.1-4.6-16.1-12.4-16.1s-12.4 6-12.4 16.1c0 10.1 4.6 16.1 12.4 16.1s12.4-6 12.4-16.1zM511 132.5h9.2v8h.2c1.3-5.1 6-8.7 11.3-8.5 1.1 0 2.2.1 3.3.4v9c-1.4-.4-2.9-.6-4.3-.6-5.3-.2-9.9 3.9-10.1 9.3 0 .5 0 1 .1 1.5v27.8H511v-46.9zM579.3 165.5c-1.3 8.5-9.6 14.3-20.2 14.3-13.6 0-22.1-9.1-22.1-23.8 0-14.7 8.5-24.2 21.7-24.2 13 0 21.1 8.9 21.1 23.1v3.3h-33.1v.6c-.6 6.7 4.4 12.6 11.1 13.2.5 0 1 .1 1.5 0 4.7.4 9.1-2.2 10.8-6.6l9.2.1zm-32.5-13.9h23.4c.3-6.2-4.4-11.5-10.6-11.9h-.9c-6.5 0-11.9 5.2-11.9 11.8v.1zM209.2 66.7c7.5-.5 14.1 5.1 14.6 12.6.1.9 0 1.8-.1 2.7 0 9.9-5.3 15.5-14.5 15.5H198V66.7h11.2zm-6.4 26.5h5.8c5.3.3 9.9-3.8 10.3-9.1 0-.7 0-1.3-.1-2 .7-5.3-3-10.2-8.3-11-.6-.1-1.2-.1-1.9-.1h-5.8v22.2zM229.1 85.9c-.6-6.1 3.9-11.5 9.9-12 6.1-.6 11.5 3.9 12 9.9.1.7.1 1.4 0 2.1.6 6.1-3.8 11.5-9.9 12.1-6.1.6-11.5-3.8-12.1-9.9 0-.8 0-1.5.1-2.2zm17.2 0c0-5-2.3-8-6.2-8-4 0-6.2 3-6.2 8 0 5.1 2.2 8 6.2 8s6.2-2.9 6.2-8zM280.3 97.6h-4.8l-4.8-17.2h-.4l-4.8 17.2h-4.7l-6.4-23.3h4.7l4.2 17.8h.3l4.8-17.8h4.4l4.8 17.8h.4l4.2-17.8h4.6l-6.5 23.3zM292.1 74.3h4.4V78h.3c1.2-2.7 4-4.4 7-4.1 4.2-.3 7.8 2.8 8.1 7 0 .6 0 1.1-.1 1.7v15.1h-4.6V83.6c0-3.7-1.6-5.6-5-5.6-2.9-.1-5.4 2.1-5.6 5.1v14.4H292l.1-23.2zM319.2 65.2h4.6v32.4h-4.6V65.2zM330.2 85.9c-.6-6.1 3.9-11.5 9.9-12 6.1-.6 11.5 3.9 12 9.9.1.7.1 1.4 0 2.1.6 6.1-3.9 11.5-9.9 12.1-6.1.6-11.5-3.9-12.1-9.9 0-.8 0-1.5.1-2.2zm17.2 0c0-5-2.3-8-6.2-8-4 0-6.2 3-6.2 8 0 5.1 2.2 8 6.2 8 3.9 0 6.2-2.9 6.2-8zM357 91c0-4.2 3.1-6.6 8.7-7l6.3-.4v-2c0-2.5-1.6-3.8-4.8-3.8-2.6 0-4.3.9-4.9 2.6h-4.5c.5-4 4.2-6.6 9.5-6.6 5.8 0 9.1 2.9 9.1 7.8v15.9H372v-3.3h-.4c-1.5 2.4-4.2 3.8-7 3.7-3.9.4-7.3-2.4-7.7-6.3.1-.1.1-.4.1-.6zm15-2v-2l-5.7.4c-3.2.2-4.7 1.3-4.7 3.4 0 2.1 1.8 3.3 4.3 3.3 3 .3 5.7-1.9 6-4.9 0-.1 0-.1.1-.2zM382.6 85.9c0-7.4 3.8-12 9.7-12 3-.1 5.8 1.5 7.1 4.1h.3V65.2h4.6v32.4h-4.4v-3.7h-.4c-1.5 2.6-4.3 4.2-7.3 4.1-5.9 0-9.6-4.7-9.6-12.1zm4.7 0c0 4.9 2.3 7.9 6.2 7.9 3.9 0 6.3-3 6.3-7.9s-2.4-7.9-6.3-7.9c-3.8 0-6.2 3-6.2 7.9zM423.3 85.9c-.6-6.1 3.9-11.5 9.9-12 6.1-.6 11.5 3.9 12 9.9.1.7.1 1.4 0 2.1.6 6.1-3.8 11.5-9.9 12.1-6.1.6-11.5-3.8-12.1-9.9.1-.8.1-1.5.1-2.2zm17.3 0c0-5-2.3-8-6.2-8-4 0-6.2 3-6.2 8 0 5.1 2.2 8 6.2 8 3.9 0 6.2-2.9 6.2-8zM451.5 74.3h4.4V78h.3c1.2-2.7 4-4.4 7-4.1 4.2-.3 7.8 2.8 8.1 7 0 .6 0 1.1-.1 1.7v15.1h-4.6V83.6c0-3.7-1.6-5.6-5-5.6-2.9-.1-5.4 2.1-5.6 5.1v14.4h-4.6V74.3zM497.2 68.5v5.9h5v3.9h-5v12c0 2.4 1 3.5 3.3 3.5.6 0 1.2 0 1.8-.1v3.8c-.8.1-1.7.2-2.5.2-5.1 0-7.1-1.8-7.1-6.3V78.3H489v-3.9h3.7v-5.9h4.5zM508.6 65.2h4.6V78h.4c1.3-2.7 4.1-4.4 7.1-4.2 4.2-.2 7.8 3 8.1 7.2 0 .5 0 1-.1 1.4v15H524V83.7c0-3.7-1.7-5.6-5-5.6-3-.2-5.6 2-5.9 5v14.5h-4.6l.1-32.4zM555.4 91.3c-1.3 4.4-5.5 7.2-10.1 6.7-5.8.2-10.7-4.5-10.9-10.3 0-.6 0-1.2.1-1.7-.8-5.9 3.3-11.3 9.2-12.1.5-.1 1-.1 1.5-.1 6.5 0 10.4 4.4 10.4 11.7v1.6h-16.4v.3c-.3 3.4 2.2 6.4 5.6 6.7h.6c2.2.3 4.4-.8 5.5-2.8h4.5zm-16.2-7.5H551c.2-3.1-2.1-5.8-5.2-6h-.5c-3.3 0-6 2.6-6 5.9-.1 0-.1 0-.1.1z"/></svg>
```

Comment:
I think this file should be actually called "light", because the button itself is white.